src/Nethermind/Nethermind.Evm/EvmStack.cs

```diff
@@ -18,6 +19,7 @@ namespace Nethermind.Evm;
 
 using Word = Vector256<byte>;
 using HalfWord = Vector128<byte>;
+using static Unsafe;
```

Comment:
I don't like this change to be honest, I would prefer each unsafe operation to be explicitly marked in code. Makes me analyze them more, now I can loose them easier.